src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -230,48 +230,48 @@ LedgerConsensusImp::LedgerConsensusImp (
         LocalTxs& localtx,
         LedgerMaster& ledgerMaster,
         FeeVote& feeVote)
-    : app_ (app)
```

Comment:
@JoelKatz Almost no time at all, because I've gotten used to it over the last 3 years that we've been naming them that way :-)